### ISSUE TITLE
Can bypass chair blocking by disabling combat mode

### ISSUE THREAD

**Issue by Rustybuckets6601** [tgstation/tgstation#69128]
Reporting client version: 514.1575

## Round ID:
188288

## Testmerges:
- [Check for compliant JSON before receiving payload](https://www.github.com/tgstation/tgstation/pull/69023)

## Reproduction:
Take one or two chairs, then have a partner attack you/shove you with combat mode on, it will be blocked. Have the same partner disable combat mode then shove or attack, it will never be blocked.

**Comment by timothymtorres** [comment]
Can you post a video of this or something?  Are you saying that the RNG fails every time for shoving?

**Comment by Rustybuckets6601** [comment]
> Can you post a video of this or something? Are you saying that the RNG fails every time for shoving?

Yes, if you disable combat mode and attack someone who is using a chair to block your attacks, you will always hit them and they will never block. I can't get a video at the moment

**Comment by Mooshimi** [comment]
exploit bc i sould like if you're in friendly mode you could avoid block-chances

**Comment by Sealed101** [comment]
fixed by https://github.com/tgstation/tgstation/pull/75759 or https://github.com/tgstation/tgstation/pull/79721
